extension/ui/components/input_bar/InputBar.tsx

```diff
@@ -269,7 +269,7 @@ export function AssistantInputBar({
     const newFiles = getFileBlobs().map((cf) => ({
       title: cf.filename,
       fileId: cf.fileId,
-      url: cf.publicUrl,
+      url: cf.sourceUrl,
```

Comment:
So changes here were to fix and allow to open directly the sourceUrl which will be the Notion page for a notion attachment (in a new tab) or would download the content if let's say we used the button to extract screenshot from the tab. 

I'm not 100% sure what this could break 😕